### PR TITLE
Exposed gizmo color alpha

### PR DESCRIPTION
**Pull request by kpal81xd**
Exposes color alpha for gizmo mesh color

### Preview
image

### CODE REVIEW

File: extras/gizmo/transform-gizmo.js [hunk 1]
```diff
@@ -430,10 +439,36 @@ class TransformGizmo extends Gizmo {
...
+     * The color alpha for all axes.
+     *
+     * @type {number}
+     */
+    set colorAlpha(value) {
+        this._colorAlpha = math.clamp(value, 0, 1);
+
+        this._updateAxisColor('x', this._meshColors.axis.x);
+        this._updateAxisColor('y', this._meshColors.axis.y);
+        this._updateAxisColor('z', this._meshColors.axis.z);
```
Comment (slimbuck) [inline extras/gizmo/transform-gizmo.js:452]: missing xyz & face?

Comment (kpal81xd) [reply extras/gizmo/transform-gizmo.js:452]: This should do all of them but you will need to set it for each of the Translate Rotate and Scale gizmo class instances

Comment (kpal81xd) [reply extras/gizmo/transform-gizmo.js:452]: Oh you mean the center igy

Comment (kpal81xd) [reply extras/gizmo/transform-gizmo.js]: Okay added 👍

**Review by slimbuck — APPROVED** [review]

**Pull request merged**